src/helpers/colorToCardDropShadow.ts

```diff
@@ -0,0 +1,17 @@
+import { dropShadow } from 'classnames/tailwind'
+import CardColor from 'models/CardColor'
+
+export default function (color: CardColor) {
```

Comment:
I think it's `colorToDropShadow.ts`, simply extend `models/Color.ts` and extend `helpers/colorToDropShadow.ts` so we don't create new files